comictalker/talkers/comicvine.py

```diff
@@ -150,27 +151,24 @@ class CVResult(TypedDict, Generic[T]):
     version: str
 
 
-CV_STATUS_RATELIMIT = 107
```

Comment:
It's no longer listed in the [API doc](https://comicvine.gamespot.com/api/documentation#toc-0-0) and as mentioned, they use the default HTTP code now.

Reply:
I meant the variable not the value

Reply:
The var isn't used because it's now a HTTP status code like 200, 500, 404 etc.. That value was used after a CV had returned something. Now, it has moved from `_get_cv_content` to `_get_url_content`.

Reply:
Then please use `requests.status_codes.codes.TOO_MANY_REQUESTS`. Here is the rationale when this was originally added https://github.com/comictagger/comictagger/pulls?q=is%3Apr+is%3Aclosed+talker#discussion_r998881387
None of the numbers should be used be used but that is outside the scope of this PR, and 200, 500 and 404 are the most common and 404 is literally a meme at this point because everyone has used a dead link before. I don't think anyone that has ever used an http library has any doubt as to what those 3 mean; substantially less people know that 429 is the proper HTTP code for too many requests.